mods/cnc/maps/scb30ea/scb30ea-AI.lua

```diff
@@ -0,0 +1,288 @@
+
```

Comment:
Missing header.

Reply:
Sorry, not sure of this... Is the copyright comment what you refer to?

Reply:
Yes. The same thing all the other lua files have.